goon/code/datums/browserOutput.dm

```diff
@@ -59,10 +59,11 @@ var/list/chatResources = list(
 		for(var/asset in global.chatResources)
 			owner << browse_rsc(file(asset))
 
-		owner << browse(file("goon/browserassets/html/browserOutput.html"), "window=browseroutput")
-		sleep(20 SECONDS)
-		if(!owner || loaded)
-			break
+		for(var/subattempts = 1 to 3)
```

Comment:
I have a hard time believing that changing `=` to `in` makes the slightest difference for either of those. :eyes: